prereise/cli/helpers.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import os
+from datetime import datetime
+
+from prereise.cli.constants import DATE_FMT
+
+
+def validate_date(date_string):
+    """Helper function to validate date strings
+
+    :param str date_string: date in string format to be validated
+    :return: (*str*) validated date string
+    """
+    datetime.strptime(date_string, DATE_FMT)
+    return date_string
+
+
+def validate_file_path(file_path):
+    """Helper function to validate file paths
+
+    :param str file_path: path include filename for where
+        to save the file
+    :raises ValueError: if the provided file path is not accessible
+    :return: (*str*) validated file path
+    """
+    if not os.access(os.path.dirname(file_path), os.W_OK):
+        raise ValueError("Please choose a valid file path")
+    if os.path.isdir(file_path):
+        raise ValueError("Please provide a filename with a .pkl extension")
+    if os.path.isfile(file_path):
+        logging.warning("File exists! Will overwrite file at end of download!")
+    return file_path
+
+
+def add_data_source_to_download_parser(data_source, parser):
+    """Helper function that adds an additional source of data
+    that the user can interact with and download via the
+    command line tool
+
+    :param class:`prereise.cli.data_sources.DataSource` data_source: an
+        instance of a class that implements the abstract class DataSource
```

Comment:
Sphinx is ale to parse `:param prereise.cli.data_sources.DataSource data_source: description` and this is what we use throughout the code base

Reply:
Great. Thanks for the catch. I wonder if there's a linter that could be added that checks/lints these docstrings, I feel like that would save some work hunting malformed docstrings 😅

Reply:
That would be great. I have seen some linter for the Google and Numpy style. For some rason we decided to use the more condensed Sphinx style.